includes/Core/Util/Entity_Factory.php

```diff
@@ -484,4 +491,73 @@ private static function prefix_title( $title, $prefix ) {
 			$title
 		);
 	}
+
+	/**
+	 * Gets the entity for the given URL, if available.
```

Comment:
@kuasha420, could you, please, provide a more relevant description for this method? After reading the description, it should be clear what this method does without reading the code itself. No need to go into too many details though, just a brief overview.